docs/en/changelogs/index.md

```diff
@@ -62,8 +62,4 @@ For information on how to upgrade to newer versions consult the [upgrading](/ins
  * [2.4.0-beta1](beta/2.4.0-beta1) - 29 January 2010
  * [2.4.0-alpha1](alpha/2.4.0-alpha1) - 11 November 2009
  * [2.3.11-rc1](rc/2.3.11-rc1) - 31 January 2011
- * [2.3.10-rc2](rc/2.3.10-rc2) - 20 December 2010
```

Comment:
reason for removing these? the 2.3 history in the changelog should exist in the 2.4 docs too.